assets/css/style.css

```diff
@@ -0,0 +1,281 @@
+/** Import Font **/
+
+@import url("https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i");
+
+/** Import Font  End **/
+
+/** Shared Button Effects **/
+.welcome-section {
+  padding-top: 50px;
+  height: 100%;
+  display: block;
+}
+
+.btn-see-project {
+  border: 1px solid #396df2;
+  padding: 7px;
+  border-radius: 8px;
+  font-size: 16px;
+  font-weight: 300;
+  text-transform: capitalize;
+  font-family: 'Poppins', sans-serif;
+  transition-duration: 0.4s;
+}
+
+a.btn-see-project {
+  text-decoration: none;
+  color: #4053fc;
+}
```

Comment:
- [ ] Kindly make sure you stick the styling of the **` See Source && See live`** buttons of the popup to the ones from the Figma template as on medium scene(Desktop) they show some styling issues.
Here is how they look:
![pop](https://user-images.githubusercontent.com/83042742/173959622-95de27ae-ee48-45b4-948b-22843918e737.png)